main.cpp

```diff
@@ -345,6 +199,52 @@ Thing 1) Guitar
     2) play a solo
     3) connect to amplifier
  */
+struct Guitar
+{
+    // number of strings
+    unsigned int numberOfStrings = 6;
+    // body wood type (string)
+    std::string boydWoodType = "Mahogany";
+    // brand name (string)
+    std::string brandName = "Jackson";
+    // body shape (string)
+    std::string bodyShape = "Default";
+    // size (char)    
+    char size = 'M'; // S, M, L, XL
+
+    // 5) nested class
+    struct GuitarString
```

Comment:
I would call this class `String`.  That way, if you create an instance in a function that is not part of the `Guitar` class, you'll be writing `Guitar::String` instead of `Guitar::GuitarString`.
When you pick names for your classes and variables, think about how your code will read when you use those names.
- [ ] revise